docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -59,9 +93,30 @@ For information on using `--tunnel-identifier` in the tunnel pool, see [High Ava
 Your ID must be ASCII.
 :::
 
+:::warning
+This option is aliased to `--tunnel-name` in versions starting from v4.7.0.
+:::
```

Comment:
Are we deprecating `--tunnel-identifier`?

Reply:
yup, was kind of hesitant adding it, but that's what we are doing